src/main/java/org/broadinstitute/hellbender/utils/io/FeatureOutputStream.java

```diff
@@ -47,11 +50,15 @@ public FeatureOutputStream( final GATKPath file,
         Utils.nonNull(dict);
         this.encoder = encoder;
         if (IOUtil.hasBlockCompressedExtension(file.toPath())) {
-            outputStream = new PositionalOutputStream(
-                            new BlockCompressedOutputStream(file.toString(), compressionLevel));
+            final BlockCompressedOutputStream bcos =
```

Comment:
For posterity, can you add a comment explaining the necessity for `BlockCompressedOutputStream` instead of `PositionalOutputStream`?